.github/workflows/ci_checks.yml

```diff
@@ -154,7 +154,7 @@ jobs:
           script: |
             adb root
             mkdir -p android-app/app/build/outputs/test-artifacts
-            android-app/gradlew -p android-app --build-cache --no-daemon -PmanifestEndpoint=${{ steps.start-simple-server.outputs.heroku_app_url }}api/ installQaDebug installQaDebugAndroidTest
```

Comment:
What's the reason we did this? 🤔

Reply:
@msasikanth This is just for testing. I won't merge this PR.

Reply:
Cool, marking this PR as draft them